cluster/gce/util.sh

```diff
@@ -744,6 +744,17 @@ function check-cluster() {
       local elapsed=$(($(date +%s) - ${start_time}))
       if [[ ${elapsed} -gt ${KUBE_CLUSTER_INITIALIZATION_TIMEOUT} ]]; then
           echo -e "${color_red}Cluster failed to initialize within ${KUBE_CLUSTER_INITIALIZATION_TIMEOUT} seconds.${color_norm}" >&2
+          if [[ ${KUBE_TEST_DEBUG-} =~ ^[yY]$ ]]; then
+            local tmp_log="$(mktemp)"
+            local file
+            for file in /var/log/startupscript.log /var/log/kube-apiserver.log; do
+              echo "${file} contents:"
```

Comment:
may also be worth noting these are coming from the master, since it's not obvious otherwise?